src/Materials/babylon.imageProcessingConfiguration.ts

```diff
@@ -2,6 +2,7 @@ module BABYLON {
     /**
      * Interface to follow in your material defines to integrate easily the
      * Image proccessing functions.
+     * @ignore

```

Comment:
@deltakosh do you have any opinion on how to handle these validation errors in the future. Currently we have been ignoring them as they are already all uppercase and modifying them to cammelcase to pass validation might brake back compat,

Reply:
We will ignore like that:)
All defines do not need doc as they are internal tools mostly